src/commands/dev.ts

```diff
@@ -240,7 +265,41 @@ export async function command(commandOptions: CommandOptions) {
         }
         return `/web_modules/${spec}.js`;
       });
+
+      if (!isModule(fileLoc)) {
```

Comment:
I could probably merge this into the above `rewrite` but have opted not to for clarity, notify me if you'd like this differently.